bucket/tuxguitar.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.5.3",
+    "description": "TuxGuitar is a multitrack guitar tablature editor and player written in Java-SWT, It can open GuitarPro, PowerTab and TablEdit files.",
+    "homepage": "http://www.tuxguitar.com.ar/",
+    "license": "GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later.html)",
+    "suggest": {
+        "JRE": "java/adopt8-hotspot-jre"
+    },
+    "architecture": {
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/projects/tuxguitar/files/TuxGuitar/TuxGuitar-1.5.3/tuxguitar-1.5.3-windows-x86.zip",
+            "hash": "sha1:390356aa044dbe332ce289b8461aa50e7afdf730",
+            "extract_dir": "tuxguitar-1.5.3-windows-x86"
+        }
+    },
+    "pre_install": "Add-Content \"$dir\\tuxguitar.ini\" \"vm.location=%JAVA_HOME%/bin/client/jvm.dll\"",
+    "bin": "tuxguitar.exe",
+    "shortcuts": [
+        [
+            "tuxguitar.exe",
+            "TuxGuitar"
+        ]
+    ],
+    "checkver": {
+        "url": "https://sourceforge.net/projects/tuxguitar/rss?path=/TuxGuitar",
+        "regex": "tuxguitar-([\\d.]+)-windows-x86.zip"
+    },
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://downloads.sourceforge.net/projects/tuxguitar/files/TuxGuitar/TuxGuitar-$version/tuxguitar-$version-windows-x86.zip",
+                "extract_dir": "tuxguitar-$version-windows-x86"
+            }
+        }
+    }

```

Comment:
```suggestion
"url": "https://downloads.sourceforge.net/projects/tuxguitar/files/TuxGuitar/TuxGuitar-$version/tuxguitar-$version-windows-x86.zip",
"extract_dir": "tuxguitar-$version-windows-x86"
```

Reply:
The same situation with the architecture. Is it okay to remove it?